### PR TITLE
Dev 4520 improve send transcation quest 1

### PR DESCRIPTION
**Pull request by nprimo**

### CODE REVIEW

File: introductions.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
+- Local networks, to test locally with a node. It is called `regtest` on Bitcoin.
 
-*🚨 Caution 🚨*
+_🚨 Caution 🚨_
 You should not use any crypto with value for any exercise of the module. If you already own crypto, we recommend that you use a new separate wallet to avoid any loss of funds. We will never use the main network of any blockchain `mainnet`.
 
 #### Mandatory
 
-1. [sendTransaction](sendTransaction/README.md) _Send a Bitcoin transaction_
+1. [sendTransaction](https://github.com/01-edu/public/blob/master/subjects/blockchain/send-transaction/README.md) _Send a Bitcoin transaction_
```
Comment (MSilva95) [inline introductions.md:20]: Why was this the only link that changed?

**Review by MSilva95 — CHANGES_REQUESTED** [review]
Just need to understand why the other links were not changed.

File: introductions.md (hunk 1 above)
Comment (nprimo) [reply introductions.md:20]: The document is not used anywhere (as far as I know) - I changed the link of this exercise only to have a PR related to 1 exercise only.

If you find it clearer, I can restore the `master` version and, in the future, update all the links in one PR.

**Review by MSilva95 — APPROVED** [review]
LGTM!

**Pull request merged**